specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -128,16 +129,42 @@ Given these elements, the number of test cases for each scenario can be calculat
 **Alternate flows**
 
 1. The skill creates a conversation (in teams) and starts a 1:1 conversation with a user in the group. Note - the 1:1 conversations created persist, and there is no way to delete them. Repeated calls to createConversation will succeed however, and return the appropriate conversationId that can be re-used.
```

Comment:
We probably need to move this description to another scenarion (see note about 3a), I would leave this scenario as a simple proactive message were the skill can only send a proactive message to a user that has already talked to the skill (so it uses ContinueConversation and does not need to create one)

Reply:
resolved